app/posts/how-to-use-cross-government-slack.md

```diff
@@ -0,0 +1,20 @@
+---
+title: How to use cross-government Slack
+description: Tips on how to join and make the most of the cross-government Slack workspace.
+date: 2023-05-19
+tags: post
+layout: post
+author:
+  name: Steve Messer
+  url: https://github.com/stevenjmesser
+image:
+  src: 
+  alt: 
+  opengraphImage: true
+---
+
+[Cross-government Slack](https://ukgovernmentdigital.slack.com/) is a Slack workspace for anyone working in central government, with channels on a range of topics. It's a place to connect with other people working on digital services across government.
+
+## How to join
+
+The workspace is open to anyone working in central government with a `.gov.uk` email address. If you're a contracter or agency working with a central government organisation, you may need to ask for a `.gov.uk` address to join the Slack.
```

Comment:
The workspace is open to anyone working in government with a `.gov.uk` email address, and other qualified public sector organisations. If you're a contractor or commercial supplier working with a public sector organisation, you will need to ask for a `.gov.uk` address (or other qualified public sector email address) before you can join this Slack Workspace.

A number of public sector email domains are able to sign-up without being invited, the list of these can be viewed via https://ukgovernmentdigital.slack.com/signup#/domain-signup.

For public sector organisations not on the email domain list, they can be invited by any existing Slack Workspace member.

Reply:
@joelgsamuel thanks! Do you know anything about the history of the cross-gov Slack that you could contribute to the post? Would be nice to set out a bit of the context, and show how it has evolved over time!

Reply:
Being cautious, I suspect that might need to be reviewed by the CDDO or CO Press Office, so not something I'm comfortable drafting. I'll check in with some folks though!